build.sh

```diff
@@ -19,18 +19,27 @@
 # * limitations under the License.
 # *
 
-#set -e # error out if required
+set -e # error out if required
 SCRIPT_EXEC="$(realpath $0)"
 MY_DIR="$(dirname $SCRIPT_EXEC)"
 
 FRAMEWORK_DIR=${MY_DIR}/framework
 UT_CONTROL_LIB_DIR=${FRAMEWORK_DIR}/ut-control
 
+# Check if the arguments contain 'TARGET=arm'
+if [[ "$*" == *"TARGET=arm"* ]]; then
+    TARGET=arm
+else
+    TARGET=linux
+fi
+echo "TARGET= [$TARGET] from [$0]"
+
+CUNIT_OBJ_DIR=${MY_DIR}/obj/${TARGET}/framework/CUnit-2.1-3
+CURL_OBJ_DIR=${FRAMEWORK_DIR}/ut-control/framework/curl/curl-8.8.0/build-${TARGET}
```

Comment:
FRAMEWORK_DIR is now the target, why do you need to specify it again?

`{FRAMEWORK_DIR}/ut-control/framework/curl/curl-8.8.0/build`